icons/r3.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>R3 icon</title><path fill="#000" d="M7.59689 15.1618h3.03191c.0682 1.2094.5962 2.1462 1.6693 2.7254.7665.4088 1.5841.4769 2.4187.2044 1.6523-.5281 2.521-2.3677 1.9248-4.0029-.5962-1.6182-2.572-2.6231-4.4627-1.6182-.1533-.2555-.2896-.4939-.4429-.7494-.3407-.5792-.6643-1.1753-1.005-1.75446-.0681-.11923-.0511-.18736.0171-.28956.7835-1.12421 1.55-2.24841 2.3165-3.37261.0341-.06813.0852-.11923.1363-.22143H8.22712c.0511-.08517.08517-.1533.11924-.2044.63023-.9198 1.26046-1.83961 1.89074-2.74237.0511-.06814.1533-.13627.2384-.13627 2.1292.01703 4.2413.01703 6.3535.01703h.1362v2.87864c0 .1022-.034.22143-.1022.32363-.6132.90277-1.2264 1.78851-1.8396 2.67424-.034.0511-.0681.11924-.1192.18737 2.027.4599 3.5259 1.58409 4.4116 3.44069.6984 1.4649.7836 2.9979.2896 4.5309-1.022 3.1171-4.2583 4.8886-7.4095 4.054-3.21931-.8176-4.70121-3.7814-4.59901-5.9447zM0 3.03617h3.04897v.78354c.3577-.1533.68134-.3066 1.022-.44287.6132-.23847 1.26047-.34067 1.92477-.3577h3.11711c.03406 0 .08516 0 .1533.01703-.06814.11924-.13627.2044-.18737.28957-.59617.8687-1.19234 1.72037-1.77147 2.58907-.1022.13627-.2044.18737-.3577.18737-.47693 0-.95387 0-1.39674.06813-1.4308.2555-2.46983 1.533-2.46983 2.99787v5.97872H.03407C0 11.127 0 7.09011 0 3.03617zM20.934 13.6111c0-.8516.6813-1.5159 1.55-1.5159.8347 0 1.516.6983 1.516 1.533-.017.8346-.6984 1.5159-1.533 1.5159-.8687.0171-1.55-.6813-1.533-1.533z"/></svg>
```

Comment:
This isn't passing the linter because there's a `fill` attribute on the path. If you remove that, it should work.

Was the precision of 5 necessary for this icon?

Reply:
I have removed the `fill` attribute but it is still not passing the linter.
Can you check the SVG code once by yourself?